phys/module_ra_rrtmg_lw.F

```diff
@@ -11981,23 +11962,28 @@ SUBROUTINE RRTMG_LWRAD(                                        &
 ! All fields are ordered vertically from bottom to top
 ! Pressures are in mb
 !
-! Annual function for co2 in WRF v4.2
-      co2 = (280. + 90.*exp(0.02*(yr-2000)))*1.e-6
 
 !ccc Read time-varying trace gases concentrations and interpolate them to run date.
 !
-#ifdef CLWRFGHG
-
-   CALL read_CAMgases(yr,julian,"RRTMG",co2,n2o,ch4,cfc11,cfc12)
-
-   IF ( wrf_dm_on_monitor() ) THEN
-     WRITE(message,*)'CAM-CLWRF interpolated values______ year:',yr,' julian day:',julian
-     call wrf_debug( 100, message)
-     WRITE(message,*)'  CAM-CLWRF co2vmr: ',co2,' n2ovmr:',n2o,' ch4vmr:',ch4,' cfc11vmr:',cfc11,' cfc12vmr:',cfc12
-     call wrf_debug( 100, message)
-   ENDIF
+   IF ( GHG_INPUT .EQ. 1 ) THEN 
```

Comment:
Are we still considering updating the value once per day?

Reply:
The amount of time it takes to interpolate two numbers (especially when compared to the RADIATION SCHEME) is minimal. My opinion, we interpolate each time we are in the radiation scheme and not worry about extra logical infrastructure that will certainly cause us restart troubles down the road. Granted, the input data is yearly or even 10-yearly, so you need an 8-byte value to even see the difference between time steps.

Reply:
If we update GHG values once per day, we will need 'save' in the declaration of those variables. I am concerned that this will break openMP as I saw before. But I am not an expert on openMP. Dave can clarify whether this is true or not.